src/styles/globals.css

```
@@ -0,0 +1,58 @@
+@font-face {
+    font-family: 'Pretendard';
```

Stylelint (17.10.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)